netlify.toml

```diff
@@ -2,3 +2,8 @@
   from = "/*"
   to = "/index.html"
   status = 200
+
+[[redirects]]
```

Comment:
https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file
> The redirects engine will process the first matching rule it finds, reading from top to bottom.